client/src/components/BizCard/BizCard.jsx

```diff
@@ -123,6 +124,7 @@ export default function BizCard({ props }) {
               src="https://www.japan-guide.com/g9/2005_01b.jpg"
             />
           </div>
+          <BizCard_Map props={biz}/>
```

Comment:
`props` should just be called `biz` so it's clear what the data is....otherwise, your props is like this `props: {props: biz}`
```suggestion
          <BizCard_Map biz={biz}/>
```

Reply:
Thx! I changed it.